eduNEXT/xblock-imagesgallery
# fix: move static content import inside the specific method

### Description
This PR fixes a runtime error due to a circular import. This caused XBlock not to be listed among the advanced components. These changes were tested in a remote environment with Redwood.

### How to Test
1. Install the XBlock with these changes in your Redwood environment.
2. Go to a unit in your course and select **Advanced**. You should see the Images Gallery component listed.

    ![image](https://github.com/user-attachments/assets/d463a259-bb0d-4064-83cb-89f5b8114166)
